### PR TITLE
Laravel 4.2 sample app with both apache and nginx/fpm docker files

### PR DESCRIPTION
**Pull request by labbati**
The very first commit is the Laravel application as created by the composer command in the getting started guide of the framework.

The real changes are here: https://github.com/DataDog/trace-examples/compare/dd056475f49f6723e25697c0792b5d0eff7f04ee...949eea77e9314dac6a658165c649e901e4afa8cf

### CODE REVIEW

**Comment by labbati** [comment]
This is a very good idea. The others required a little bit more work, so I took the shortcut 😄 but I want to do it because it makes a lot of sense.
Thanks!

**Comment by labbati** [comment]
@pawelchcki I based the images on `php:5.6-php` and performed some refactoring. So basically what's in `php/Laravel42` folder is pretty much the app, while everything in `php` it may be worth a more in depth code review. Note that the file `php/apache2.conf` is almost a copy and paste of the one in the image, with the exception of the `DocumentRoot` param.

It would be great to have the final round of CR and merge it!
Thanks

File: php/Laravel42/app/routes.php [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+|--------------------------------------------------------------------------
+|
+| Here is where you can register all of the routes for an application.
+| It's a breeze. Simply tell Laravel the URIs it should respond to
+| and give it the Closure to execute when that URI is requested.
+|
+*/
+
+Route::get('/', ['uses' => 'HomeController@showWelcome', 'as' => 'simple_route_name']);
+Route::get('/distributed-tracing', 'HomeController@distributedTracing');
```
Comment (SammyK) [inline php/Laravel42/app/routes.php]: I get an Apache-404 page when I try to visit this URL. I'm wondering if we're missing the Apache config for mod_rewrite? We should probably mention this route in the README, or was this just a left over test? :)

Comment (labbati) [reply php/Laravel42/app/routes.php]: Probably I left mod_rewrite config out when moving from the preconfigured image to the one based on `php` base images. Thanks for noticing. Looking at it

File: php/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+## Welcome to the php tracer sample apps
+
+### Laravel 4.2 APM sample app
+
+Start up an agent: you have to set the env `DATADOG_API_KEY` with your own DD api key.
+
+    $ docker-composer up -d agent
```
[the remark this answers is not in the document]
Comment (labbati) [reply php/README.md]: 😄

File: php/README.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+Build the image you are interested in:
+
+    # Php 5.6 - Apache - Laravel 4.2
+    $ docker-compose build laravel42_php56_apache
+
+    # Php 5.6 - Nginx/fpm - Laravel 4.2
+    $ docker-compose build laravel42_php56_nginx
```
[the remark this answers is not in the document]
Comment (labbati) [reply php/README.md]: Right, to be removed, was there before. Good catch

File: php/README.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+Run the image you intend to try out:
+
+    # Php 5.6 - Apache - Laravel 4.2
+    $ docker-compose run --rm laravel42_php56_apache composer update
+    $ docker-compose up laravel42_php56_apache
+
+    # Php 5.6 - Nginx/fpm - Laravel 4.2
+    $ docker-compose run --rm laravel42_php56_nginx composer update
+    $ docker-compose up laravel42_php56_nginx
```
[the remark this answers is not in the document]
Comment (labbati) [reply php/README.md]: 🙇

**Pull request merged**